recipe/run_test.py

```diff
@@ -1,28 +1,35 @@
+import subprocess
 from dateutil.relativedelta import relativedelta
 from dateutil.easter import easter
 from dateutil.rrule import rrule, YEARLY, FR
 from dateutil.parser import parse
 from dateutil.tz import tzlocal
+from dateutil import zoneinfo, utils
+
+
+# Test pip check
+subprocess.run(["pip", "check"])
+
```

Comment:
Could you add a test to check version?

Reply:
Artifacts and rendered `meta.yaml` have `v2.8.3a1` but if I unpack the artifact it has inside `site-packages`:
-  and `_version.py` file in the `dateutils` folder
-  the folder `python_dateutil-2.8.2.dist-info` with the `METADATA` file which contains `Version: 2.8.2`

Reply:
I applied a patch to use the version `2.8.3+snowflake1`. Let me know if my attempt isn't correct. 
I'm not fixing the version as 2.8.**2**+snowflake1 because I'm not sure if end-users will take it instead of the already existing `v2.8.3a1`.

Reply:
If our solver respects alpha+beta, `2.8.2+snowflake1` should be taken unless the user explicitly requests `v2.8.3a1`. Pip has a `--pre` flag that can be used to ask the solver to prefer prereleases over normal releases, but I don't think conda has this concept...

In all cases, I think it's fine if we use `2.8.3+snowflake1`.

Reply:
Also, we could maybe check if we have a concept of yanking versions (https://pypi.org/help/#yanked). Because if we have, we could simply yank `2.8.3a1` and it wouldn't be resolved anymore unless explicitly requested (no matter what the solver does).

Reply:
The whole point of yanking (at least on PyPI is that it doesn't break anyone. It just makes it so that people can't mistakenly install the yanked version unless they use `==`.

Reply:
> Could you add a test to check version?

@ryanskeith Done!